homeworks/andrey.makarenko_Baksckua/3-functions/functions.js

```diff
@@ -0,0 +1,16 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+
+const numberOfTanksOil = 3;
+const numberOfTanksWeter = 5;
+const numberOfTanksMerkury = 2;
+
+const calcSumOil = OIL * numberOfTanksOil;
+const calcSumWeter = WATER * numberOfTanksWeter;
+const calcSumMerkury = MERCURY * numberOfTanksMerkury;
+
+const getWeight = function () {
```

Comment:
Please, review how the `getWeight` function should be in the homework task. It should take 3 arguments `weight, liquid, count` and work like this:

```js
const OIL = 926;

const numberOfTanksOil = 3;

const volume = 1000; // the volume of the tank

const oilWeight = getWeight(volume, OIL, numberOfTanksOil);

function oilWeight(weight, liquid, count) {
  // some logic
}
```